package.json

```diff
@@ -60,14 +59,6 @@
     "simple-git-hooks": ">=2.8.0",
     "typescript": "4.7.2"
   },
-  "peerDependencies": {
-    "node-notifier": ">=9.0.1"
```

Comment:
We do have dependency on "node-notifier" in "test-79-npm" at the moment.

Reply:
Yes, that's what I was trying to see in the first commit, and all the tests seems to pass.

It confirms my suspicion with the `"peerDependenciesMeta"` that this package won't be installed unless specified explicitly, so it's not installed even before this change, which we can confirm by looking at the lockfile not having `"node-notifier"` anywhere.

Another thing is that "test-79-npm" seems to download all the packages by itself using `https` module, which might not be the best approach to testing these kinds of stuff, but that would also mean we may not need any of those packages in the project's `package.json`.